src/components/MySongsTable/MySongsTable.js

```diff
@@ -0,0 +1,15 @@
+import mySongsTableTemplate from "./MySongsTable.html";
+export class MySongsTableComponent {
```

Comment:
please add space between import declaration and class decl

Reply:
resolved